From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tri Vo <trong@google.com>
Date: Thu, 16 May 2019 09:57:57 -0700
Subject: ANDROID: gki_defconfig: remove more recommended configs

Disable following configs:
CONFIG_KSM
CONFIG_SGETMASK_SYSCALL
CONFIG_ARM64_SW_TTBR0_PAN

Bug: 132717553
Test: boot arm64 cuttlefish
Signed-off-by: Tri Vo <trong@google.com>
Change-Id: I2689515f3ddff10d6ad09c41ac5f4a0415922357
---
 arch/arm64/configs/gki_defconfig | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 9c1086a740cf..b7ae05eb896e 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -22,7 +22,6 @@ CONFIG_BLK_DEV_INITRD=y
 # CONFIG_RD_XZ is not set
 # CONFIG_RD_LZO is not set
 # CONFIG_RD_LZ4 is not set
-CONFIG_SGETMASK_SYSCALL=y
 # CONFIG_SYSFS_SYSCALL is not set
 # CONFIG_FHANDLE is not set
 CONFIG_KALLSYMS_ALL=y
@@ -40,7 +39,6 @@ CONFIG_ARMV8_DEPRECATED=y
 CONFIG_SWP_EMULATION=y
 CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_SETEND_EMULATION=y
-CONFIG_ARM64_SW_TTBR0_PAN=y
 CONFIG_RANDOMIZE_BASE=y
 # CONFIG_EFI is not set
 CONFIG_COMPAT=y
@@ -67,7 +65,6 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_MODVERSIONS=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 # CONFIG_SPARSEMEM_VMEMMAP is not set
-CONFIG_KSM=y
 CONFIG_TRANSPARENT_HUGEPAGE=y
 CONFIG_ZSMALLOC=y
 CONFIG_NET=y
